addons/l10n_sa/__init__.py

```diff
@@ -2,7 +2,18 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from odoo import api, SUPERUSER_ID
+from odoo.tools import load_language
+from . import models
+
+
+def pre_init_hook(cr):
+    load_language(cr, lang="ar_001")
+
+
+def post_init_hook(cr, registry):
+    load_translations(cr, registry)
```

Comment:
other l10n loaded translations this way (be, id, ca, ch, it), however the wrapper function is useless, i changed it so that load_translations is called directly in the manifest